src/routes/api/comments/[referenceType]/[referenceId]/index.ts

```diff
@@ -79,7 +79,7 @@ export const POST = authenticated(async (event) => ({
 	body: await run(async (client) =>
 		createComment(
 			client,
-			event.locals.token.userId,
+			event.locals.token?.userId,
```

Comment:
Why is this optional here? Is this just a pleasing TypeScript thing b/c locals is an any object or something? because we should definitely disallow the possibility of creating a comment without the user's Id?

Reply:
I guess this would throw an error any way though right? B/c calling (undefined/null).userId would throw a TypeError and so we'd never create the comment.